osu.Game/OsuGameBase.cs

```diff
@@ -90,6 +91,10 @@ public OsuGameBase()
         protected override IReadOnlyDependencyContainer CreateLocalDependencies(IReadOnlyDependencyContainer parent) =>
             dependencies = new DependencyContainer(base.CreateLocalDependencies(parent));
 
+        protected virtual void PostNotification(Notification notification)

```

Comment:
It seems to me like this is needed because `notificationOverlay` only exists in `OsuGame`, but `BeatmapManager` exists in `OsuGameBase`. This seems a bit weird, though. Isn't a notification system more "basic" than osu!'s beatmap system? Can you remind me what the distinction between `OsuGameBase` and `OsuGame` was for again?